### ISSUE TITLE
Cancelled Orders not showing as Cancelled

### ISSUE THREAD

**Issue by jefhar** [Attendize/Attendize#234]
I have orders showing as "Awaiting Payment" even though the order has been cancelled. The tickets are showing as cancelled, but the order is still 'Awaiting Payment.'

```
]> select orders.id 'orders.id', orders.order_status_id 'orders.order_status_id', order_statuses.name 'order_statuses.name', orders.is_cancelled 'order.is_cancelled', attendees.is_cancelled 'attendees.is_cancelled' from orders, order_statuses, attendees where orders.id=attendees.order_id and order_status_id=order_statuses.id and orders.id=14;
+-----------+------------------------+---------------------+--------------------+------------------------+
| orders.id | orders.order_status_id | order_statuses.name | order.is_cancelled | attendees.is_cancelled |
+-----------+------------------------+---------------------+--------------------+------------------------+
|        14 |                      5 | Awaiting Payment    |                  0 |                      1 |
|        14 |                      5 | Awaiting Payment    |                  0 |                      1 |
+-----------+------------------------+---------------------+--------------------+------------------------+
2 rows in set (0.00 sec)
```

**Comment by givre** [comment]
I confirm.

**Comment by daveearley** [comment]
I had a look into this and it looks like an order was only being considered cancelled if the attendees were explicitly being refunded and cancelled. So when the offline payments feature was added it created a few bugs. I'll work on a new process to make cancelling/refunding work more fluidly.

**Comment by etiennemarais** [comment]
Refunds and Cancellation issues are fixed in #721. Closing this
